### DISCUSSION TITLE
能分享下多语言混读的朗读规则吗？

### DISCUSSION THREAD

**Discussion by Skylarzz** [jing332/tts-server-android#226] [Q&A]
README 里面链接失效了

**Comment by Myst1cX** [comment]
Have you found a configuration?

**Reply by Skylarzz** [reply]
没有

**Reply by Myst1cX** [reply]
> 没有

You are talking about switching between languages automatically or narrator/dialogue config?

**Reply by Skylarzz** [reply]
> You are talking about switching between languages automatically or narrator/dialogue config?

多语言自动切换的配置我没有找到，旁白与对话的配置我会使用．
不过我知道 TTS Server Stable 版，Dev 版，与 MultiTTS 是可以共存的．所以我安装了多个 TTS 软件，以及多个小说阅读软件，不同语言的内容我使用不同的软件组合，勉强算是个 workaround 方法

**Reply by Myst1cX** [reply]
> > You are talking about switching between languages automatically or narrator/dialogue config?
> 
> I couldn't find the configuration for automatic multilingual switching, but I know how to use the narration and dialogue configuration. However, I know that TTS Server Stable and Dev versions can coexist with MultiTTS. Therefore, I installed multiple TTS software programs and multiple novel reading software programs, using different software combinations for different language content, which is a workaround.

Could you please share the narrator/dialogue configuration and how to get multiple TTS Server editions? I would really appreciate it

**Reply by Skylarzz** [reply]
> Could you please share the narrator/dialogue configuration and how to get multiple TTS Server editions? I would really appreciate it

TTS Server 1: https://jing332.lanzn.com/b09jpjd2d     Password: 1234
TTS Server 2: https://jing332.lanzn.com/b09ig9qla     Password: 1234
MultiTTS https://pan.baidu.com/s/1eejf0RYXGf-TiK3dFB3BAA?pwd=mfxm
KaldiTTS  https://huggingface.co/spaces/k2-fsa/text-to-speech

旁白对话的语音区分设置用软件 GUI 设置一下就好

**Reply by Myst1cX** [reply]
> > Could you please share the narrator/dialogue configuration and how to get multiple TTS Server editions? I would really appreciate it
> 
> TTS Server 1: https://jing332.lanzn.com/b09jpjd2d Password: 1234 TTS Server 2: https://jing332.lanzn.com/b09ig9qla Password: 1234 MultiTTS https://pan.baidu.com/s/1eejf0RYXGf-TiK3dFB3BAA?pwd=mfxm KaldiTTS https://huggingface.co/spaces/k2-fsa/text-to-speech
> 
> 旁白对话的语音区分设置用软件 GUI 设置一下就好

Thank you for these links! How did you add KaldiTTS to TTS Server?
